pkg/cluster/internal/create/actions/installcni/cni.go

```diff
@@ -51,43 +47,11 @@ func (a *action) Execute(ctx *actions.ActionContext) error {
 		return err
 	}
 
-	// read the manifest from the node
-	var raw bytes.Buffer
```

Comment:
removing this breaks compatibility with certain kind images for pretty much zero gain

Reply:
got it, never thought about it 😅